src/contracts/api.ts

```diff
@@ -1,33 +1,55 @@
 import { AuthenticationSettings } from "./authenticationSettings";
-import { ArmResource } from "./armResource";
 
 /**
  * Cotract of API
  */
-export interface ApiContract extends ArmResource {
-    properties: ApiProperties;
-}
+export interface ApiContract {
+    /**
+   * API identifier.
```

Comment:
nit: would be great to have an example in the comment `* API identifier` > `* API identifier, e.g. "echo-api".`